gatsby-ssr.tsx

```diff
@@ -34,3 +34,16 @@ export const wrapPageElement: GatsbySSR<any, Context>["wrapPageElement"] = ({
 
   return <Layout {...props}>{element}</Layout>
 }
+
+export const onRenderBody = ({ setHeadComponents }: RenderBodyArgs) => {
+  setHeadComponents([
+    <link
+      rel="preload"
+      href="/fonts/Inter-roman.var.woff2"
+      as="font"
+      type="font/woff2"
+      crossOrigin="anonymous"
+      key="interFont"
+    />,
+  ])
+}
```

Comment:
@pettinarip added this preload per [Gatsby docs](https://www.gatsbyjs.com/docs/how-to/styling/using-local-fonts/#preload-your-fonts)

Reply:
Nice! not sure how this works if we define multiple fonts.

Reply:
@pettinarip Since this is making a request ahead of time, there would have to be consideration on what font would be the most noticeable by the user, as doing this multiple times could impact performance.